examples/blog-with-comment/posts/riders-of-rohan.mdx

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Could this use `@next/mdx` instead? Or maybe MDX isn't even needed here - just load markdown files and parse with gray-matter for the front matter?

Reply:
I preferred next-mdx because it did not require user to configure anything. But if this is a problem for you, I can switch to @next/mdx

Reply:
GIven that it's just normal markdown, let's follow what nextjs.org/learn teaches for handling Markdown!

Reply:
ok, I will use @next/mdx